pkg/cli/mirror/catalog_images.go

```diff
@@ -195,22 +209,53 @@ func (o *MirrorOptions) processCatalogRefs(ctx context.Context, catalogsByImage
 
 		klog.Infof("Rendering catalog image %q with file-based catalog ", refExact)
 
-		add, err := builder.LayerFromPath("/configs", filepath.Join(artifactDir, config.IndexDir, "index.json"))
+		configLayerToAdd, err := builder.LayerFromPath("/configs", filepath.Join(artifactDir, config.IndexDir, "index.json"))
 		if err != nil {
 			return fmt.Errorf("error creating add layer: %v", err)
 		}
 
 		// Since we are defining the FBC as index.json,
 		// remove anything that may currently exist
-		deleted, err := deleteLayer("/.wh.configs")
+		deletedConfigLayer, err := deleteLayer("/.wh.configs")
 		if err != nil {
 			return fmt.Errorf("error creating deleted layer: %v", err)
 		}
 
-		// Delete must be first in the slice
-		// so that the /configs directory is deleted
-		// and then add back with the new FBC.
-		layers := []v1.Layer{deleted, add}
+		// white out layer /tmp
+		// TODO : any label in the manifests that might indicate more accurately where the cache is?
```

Comment:
I don't think you're going to find labels for where the cache is, but you do have the docker command that tells the image how the image should be started. 

The CMD values come from the docker file that builds the image. An example of how docker files are created for catalog images can be found in [generate_dockerfile.go](https://github.com/operator-framework/operator-registry/blob/v1.25.0/alpha/action/generate_dockerfile.go). That link is the very first release of operator-registry that contains the `--cache-dir` argument. There's also examples in operator-sdk project too, but I've not included them here.

For example:
```
ENTRYPOINT ["/bin/opm"]
CMD ["serve", "/configs", "--cache-dir=/tmp/cache"]
```

You could obtain the `cmd` value from the image config and attempt to find the `--cache-dir` argument and get its value. You should probably attempt to handle both variations where the equals sign is used versus separate arguments, just to be safe:

CMD ["serve", "/configs", "--cache-dir=/tmp/cache"]
CMD ["serve", "/configs", "--cache-dir", "/tmp/cache"]

If you process `cmd` and can't find the argument, you could just assume `/tmp` as a fallback.